### PR TITLE
en: update lightning deployment using S3 IAM

### PR DESCRIPTION
**Pull request by ran-huang**
### What is changed, added or deleted? (Required)

- Update lightning deployment using S3 IAM
- Waiting for #5 

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-tidb-operator/pull/47
- Other reference link(s):

### CODE REVIEW

File: en/restore-data-using-tidb-lightning.md [hunk 1]
```diff
@@ -130,23 +136,95 @@ You can deploy tikv-importer using the Helm chart. See the following example:
...
+
+            Fill in the placeholders with your configurations and save it as `secret.yaml`, and then create the `Secret` via `kubectl apply -f secret.yaml -n <namespace>`.
 
-        3. Configure the `dataSource.remote.storageClassName` to an existing storage class in the Kubernetes cluster.
+    3. Configure the `dataSource.remote.storageClassName` to an existing storage class in the Kubernetes cluster.
 
-2. Deploy TiDB Lightning
+### Deploy
+
+The method of deploying TiDB Lightning varies with different methods of granting permissions and different storages.
```
Comment (TomShawn) [inline en/restore-data-using-tidb-lightning.md]:
```suggestion
The method of deploying TiDB Lightning varies with different methods of granting permissions and with different storages.
```

File: en/restore-data-using-tidb-lightning.md [hunk 2]
```diff
@@ -130,23 +136,95 @@ You can deploy tikv-importer using the Helm chart. See the following example:
...
+
+        > **Note:**
+        >
+        > `arn:aws:iam::123456789012:role/user` is the IAM role created in Step 1.
+        > `<service-account>` is the ServiceAccount used by TiDB Lightning. The default value is `default`.
 
-> **Note:**
->
-> Currently, TiDB Lightning will [exit with non-zero error code even when data is successfully restored](https://github.com/pingcap/tidb-lightning/pull/230). This will trigger the job failure. Therefore, the success status needs to be determined by viewing tidb-lightning pod's log.
+When TiDB Lightning fails to restore data, it cannot simply be restarted. **Manual intervention** is required. So the TiDB Lightning's `Job` restart policy is set to `Never`.
```
Comment (TomShawn) [inline en/restore-data-using-tidb-lightning.md]:
```suggestion
When TiDB Lightning fails to restore data, you cannot simply restart it. **Manual intervention** is required. So the TiDB Lightning's `Job` restart policy is set to `Never`.
```

**Comment by TomShawn** [comment]
@ran-huang Have you updated this part?
image

**Comment by ran-huang** [comment]
> @ran-huang Have you updated this part?
> image

Yes, it's been included when I created the `restore-from-aws-s3-using-br.md` file. image

**Pull request merged**